tests/unit/test_api_citation_blueprint.py

```diff
@@ -0,0 +1,66 @@
+from flask import Flask
+import pytest
+
+from api.blueprints.drbCitation import citationFetch
+from api.utils import APIUtils
+
+
+class TestCitationBlueprint:
```

Comment:
These tests look great (see comment below for why I think they're failing). I'd add one more to test that we get the expected response if `formats=` is not set.